### PR TITLE
Add PAL refresh button

### PR DESCRIPTION
**Pull request by zfox23**
Since the PAL doesn't (and shouldn't) auto-refresh, and, currently, the only way to refresh the contents of the PAL is to close and re-open it, we should add a convenient refresh button.

Thanks to @AlanZimmerman for the styling help.

### CODE REVIEW

File: scripts/system/pal.js [hunk 1]
```diff
@@ -265,13 +269,13 @@ function onClicked() {
 //
 // Button state.
 //
-function onVisibileChanged() {
+function onVisibleChanged() {
```
Comment (zfox23) [inline scripts/system/pal.js]: Just a typo fix.

File: interface/resources/qml/hifi/Pal.qml [hunk 2]
```diff
@@ -47,7 +47,7 @@ Item {
         id: myInfo
         // Size
         width: pal.width
-        height: myCardHeight
+        height: myCardHeight + 20
```
Comment (zfox23) [inline interface/resources/qml/hifi/Pal.qml]: Height update as per Alan's request.

**Comment by zfox23** [comment]
[Add refresh to PAL](https://trello.com/c/h7jWXHnK/41-add-refresh-to-pal)

**Comment by zfox23** [comment]
capture

Spot the new button? 😄

**Comment by davidkelly** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5490dddbbf6fdb64fb4c2ca5096b23a195ed3865 available at: https://gist.github.com/ad80655d9ffc6267c164c61222aeba15<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9268-5490ddd.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9268-5490ddd.dmg

**Comment by davidkelly** [comment]
Tested by summoning avatars.  Brought PAL up first, then summoned, and refreshed while they were popping in.  Updated as expected 👍

**Pull request merged**